src/Employer/Employer.Web/Extensions/WageExtensions.cs

```diff
@@ -7,14 +8,38 @@ namespace Esfa.Recruit.Employer.Web.Extensions
 
     public static class WageExtensions
     {
-        public static string ToText(this Wage wage)
+        public static string ToText(this Wage wage, Func<Tuple<decimal, decimal>> getNationalMinimumWageRange, Func<decimal> getApprenticeNationalMinimumWage)
         {
-            if (wage.WageType == WageType.FixedWage)
+            const int WeeksPerYear = 52;
+
+            string wageText = "";
+
+            switch (wage.WageType)
             {
-                return $"£ {wage.FixedWageYearlyAmount?.AsMoney()} yearly";
+                case WageType.FixedWage:
+                    wageText = $"£{wage.FixedWageYearlyAmount?.AsMoney()}";
+                    break;
+                case WageType.NationalMinimumWage:
+                    var hourlyRange = getNationalMinimumWageRange();
+
+                    var minYearly = hourlyRange.Item1 * wage.WeeklyHours.Value * WeeksPerYear;
+                    var maxYearly = hourlyRange.Item2 * wage.WeeklyHours.Value * WeeksPerYear;
+
+                    wageText = $"£{minYearly.AsMoney()} - £{maxYearly.AsMoney()}";
+                    break;
+                case WageType.NationalMinimumWageForApprentices:
+                    var hourly = getApprenticeNationalMinimumWage();
+
+                    var yearly = hourly * wage.WeeklyHours.Value * WeeksPerYear;

```

Comment:
Would we ever need to do any rounding to the closest penny?

Reply:
👍